front_end/entrypoints/rn_inspector/rn_inspector.ts

```diff
@@ -23,10 +23,6 @@ Root.Runtime.experiments.register(
   /* unstable */ false,
 );
 
-Root.Runtime.experiments.enableExperimentsByDefault(
```

Comment:
1. Note that users can quite easily re-enable this experiment in the DevTools settings. Maybe we should instead remove the registration call in https://github.com/motiz88/rn-chrome-devtools-frontend/blob/70612c085c146589d501edfb44f5a6c0c414729e/front_end/panels/rn_welcome/rn_welcome-meta.ts#L35-L47?
2. It _might_ be useful to conceptually decouple the Welcome panel from the `REACT_NATIVE_SPECIFIC_UI` flag, or rename the flag to be more specific to the Welcome panel so we have better control.